config/config.form.yaml

```diff
@@ -152,7 +152,7 @@ sector:
   tes: false
   marginal_cost_storage: 0. #1e-4
   stores: ["H2"]
-  storage_units: ["battery"]
+  storage_units: ["battery", "iron-air battery"]
```

Comment:
I would kinda like having the nomenclature be changed to li-ion battery as I think this would increase clarity in the model overall. But indeed this could have massive implications. I will have a look at this

Reply:
I decided to change `battery` to `li-ion battery` in every place it was referenced and it seems to work fine without any other implications.